File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    var answer = 0;
    possible = ['aya', 'ye', 'woo', 'ma']
    babbling.forEach((babb) => {
        let babb_len = babb.length
        possible.forEach((word) => {
            if(babb.includes(word)){
                babb_len -= word.length
            }
        })
        if (babb_len === 0){
            answer ++;
        }
    })
    return answer;
}

File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    answer = 0
    arr = ["aya", "ye", "woo", "ma"]
    
    for i in babbling:
        flag = i
        syllables_count = 0
        for j in arr: 
            if flag.count(j) == 1:
                flag = flag.replace(j," ", 1)
                syllables_count += 1
        if flag.strip() == "" and syllables_count == len(arr):
            answer += 1
    return answer